frank-flow/src/frontend/src/app/header/header.component.scss

```diff
@@ -21,14 +21,19 @@
   width: 100%;
   height: 64px;
 
-  display: flex;
   justify-content: space-between;
   align-items: center;
   padding: 0 16px;
+  overflow: hidden;
+  display: grid;
```

Comment:
The `:host` is already a grid. Adding another one to it can get complicated in the feature.